pages/index.tsx

```diff
@@ -4,7 +4,7 @@ import Popper from "@popperjs/core"
 import Head from "next/head"
 
 import Testimonials from "../components/sections/homepage/Testimonials"
-import { Hero } from "../components/sections/homepage/Hero"
```

Comment:
notice that the Testimonials section is being imported using all lowercase Homepage, and the Hero section is now being imported with the first H capitalized.